examples/.debt_ceiling.rb.example

```diff
@@ -5,7 +5,7 @@ DebtCeiling.configure do |c|
   c.reduction_target = 100
   c.reduction_date   = 'Jan 1 2015'
   #set the multipliers per line of code in a file with each letter grade, these are the current defaults
-  c.grade_points = { a: 0, b: 10, c: 20, d: 40, f: 100 }
+  c.grade_points = { a: 0, b: 3, c: 13, d: 55, f: 144 }
```

Comment:
Yes, for fun, but also so there's some rhyme/reason to them?  Could just go with powers or 2 or something, this is a little weird as its every 3rd fibonacci number, but ...? seemed reasonable.